pyproject.toml

```diff
@@ -1,4 +1,107 @@
+[tool.poetry]
+name = "dowhy"
+version = "0.8.0"
+description = "DoWhy is a Python library for causal inference that supports explicit modeling and testing of causal assumptions"
+authors = [
```

Comment:
we can change the authors to "PyWhy community".

Reply:
Sure thing